content/en/database_monitoring/setup_oracle/rds.md

```diff
@@ -26,15 +26,23 @@ Complete the following steps to enable Database Monitoring with your database:
 1. [Grant the Agent access to the database](#grant-the-agent-access).
 2. [Install the Agent](#install-the-agent).
 
-## Grant the Agent access
+## Agent database user setup
 
 The Datadog Agent requires read-only access to the database server in order to collect samples.
 
+### Create user
+
+If you've previously installed the Oracle integration, skip this step because the user already exists. You must, however, execute the subsequent steps.
```

Comment:
```suggestion
If you installed the Oracle integration, skip this step because the user already exists. You must, however, execute the subsequent steps.
```